packages/devtools_app/integration_test/test/live_connection/eval_and_browse_test.dart

```diff
@@ -2,17 +2,22 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// Do not delete these arguments. They are parsed parsed by test runner.
-// test-argument:experimentsOn=true
+// Do not delete these arguments. They are parsed by test runner.
 // test-argument:appPath="test/test_infra/fixtures/memory_app"
+// test-argument:experimentsOn=true
 
 // ignore_for_file: avoid_print
 
+import 'package:devtools_app/src/screens/memory/panes/control/primary_controls.dart';
+import 'package:devtools_app/src/screens/memory/panes/diff/widgets/snapshot_list.dart';
+import 'package:devtools_app/src/shared/banner_messages.dart';
+import 'package:devtools_app/src/shared/common_widgets.dart';
```

Comment:
consider importing devtools_app.dart in tests instead of importing everything individually

Reply:
Not everything is available via devtools_app.dart and we want to be consistent. So, I suggest to keep individual imports the same way as in application.

Reply:
Using devtools_app.dart is fine in tests, but we do not need to enforce it. In tests it is fine to use either or 👍